package.json

```diff
@@ -58,6 +58,6 @@
     "typescript": "^3.9.3"
   },
   "peerDependencies": {
-    "electron": "^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0 || ^16.0.0"
```

Comment:
I see the devDependencies were not updated for a long time. Could you update that one too?

Reply:
Update to 17.0.0 I guess.

note: I'm talking about the one in devDependencies, not peerDependencies